Resources/pt-BR.json

```diff
@@ -0,0 +1,2395 @@
+{
+   "pl":{
```

Comment:
@juniorgarcia Shouldn't it be "pt-BR"?

Reply:
Yes! Sorry, I copied from another file and forgot to change.